project/src/components/homePage.jsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Card from '../components/card.jsx';
+
+function HomePage(props) {
+  const { rentalOfferCout } = props;
+  const { cardsCout } = props;
+  const cards = Array(cardsCout).fill('').map((card, i) => <Card key={i.toString()} card={card} />);
```

Comment:
```suggestion
  const cards = new Array(cardsCout).fill('').map((card, i) => <Card key={i.toString()} card={card} />);
```
Лучше всегда указывай new перед вызовами конструктора

А такой вариант, как по мне читаемее:
```suggestion
{
 Array.from(new Array(cardsCout), (card, i) => <Card key={i.toString()} card={card} />));
}
```
(его нужно делать прям внутри jsx)